modules/pam_mkhomedir/mkhomedir_helper.c

```diff
@@ -29,6 +29,8 @@
 static unsigned long u_mask = 0022;
 static unsigned long home_mode = 0;
 static char skeldir[BUFSIZ] = "/etc/skel";
+static const char dir_mode_option[] = "copymode";
+static int dir_mode = 0;
```

Comment:
If `dir_mode` is just a flag, let's reflect it in its name, e.g. `in_copy_mode` or `use_copy_mode` or something like this.

Reply:
I like `use_copy_mode` , thanks for the propositions.